### PR TITLE
docs: augment examples in readme

### PR DESCRIPTION
**Pull request by dgschwindturo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,22 +5,55 @@ configure credentials for any utility that uses the git command, or any utility
...
 
+```yaml
+jobs:
+    checkout:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Authorize
+              uses: open-turo/action-git-auth@v2
+              with:
+                  server: open-turo
```
Comment (shakefu) [inline README.md:40]:
```suggestion
                  server: githubenterprise.examplecompany.com
```

File: README.md [hunk 2]
```diff
@@ -5,22 +5,55 @@ configure credentials for any utility that uses the git command, or any utility
...
+```yaml
 jobs:
     checkout:
         runs-on: ubuntu-latest
         steps:
             - name: Authorize
               uses: open-turo/action-git-auth@v2
               with:
+                  server: open-turo
+                  prefix: xyz
```
Comment (shakefu) [inline README.md:56]:
```suggestion
                  server: git.example.com
                  prefix: github-org
```

**Review by shakefu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dgschwindturo) [reply README.md]: Thanks, done.

File: README.md (hunk 2 above)
Comment (dgschwindturo) [reply README.md]: Addressed this too thanks

**Review by shakefu — APPROVED** [review]

**Pull request merged**